pennylane/plugins/__init__.py

```diff
@@ -22,7 +22,9 @@
     :toctree: api
 
     default_qubit
+    default_qubit_tf
     default_gaussian
+    tf_ops
 """
 from .default_qubit import DefaultQubit
 from .default_gaussian import DefaultGaussian
```

Comment:
Should we import the new plugin here?

Reply:
Better that we don't, since the import depends on TensorFlow being installed. This can cause issues for users without TF